### ISSUE TITLE
RNA (Roman Numerals) not reading on notes with screen reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31447]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When you arrow along notes that have Roman Numerals attached the screen reader no longer announces the value of the Roman Numeral, it just says Roman Numeral.
When you alt arrow on to the Roman Numeral itself the speech is now very verbose.
This was working correctly a few months ago so has changed somewhere in the 4.6 updates.

STEPS:
1.  Write a few notes and add a Roman numeral to each.
2. With screen reader on arrow along the notes and note that after it announces the note/bar/beat, etc it just says "Roman Numeral" it used to announce what the Roman numeral is e.g. Roman Numeral four, roman numeral two b, etc.
3. Now Use alt arrow to move on to the Roman Numeral itself and instead of hearing Roman numeral four" or "Roman numeral iib",etc you get "Roman Numeral bar 2 beat 1 Edit two back slash b".  

This is incredibly difficult to use when working with choral harmony as arrowing along the Bass line no longer announces the Roman Numerals at all and the output when you do alt arrow on to them can be difficult to navigate amongst multiple voices on a staff as well as giving incredibly verbose output that does not actually speak the name of the chord as expected and is hard to hear for example V or b in a string of words when it should have announced Five for V.  This is making this kind of work almost impossible while trying to get this info and memorise all the parts above! 

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.4

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jrbowden** [comment]
I confirm this is a problem.
I think RNA should be announced in a very similar way as for other kinds of text objects, including stave text, figured bass, lyrics, chord symbols, etc, etc.

**Comment by jmwmusic** [comment]
To clarify what the expected speech output would be for these.
It should read them as a person reading it out would, so IVb should say "four b", V7c would say "five seven c" and symbols for flat, diminished, etc should also speak their name correctly.  Currently when Alt arrowing onto the Roman numeral element they are reading the characters entered into the box so saying "i i back slash b" for example instead of "ii flat".  It would also be helpful if they spoke whether they are capitals or not when you Alt arrow onto it for those specifying Major/minor in this way, this could just say "Cap" before the first letter.  But the most important thing is that it speaks the name of the roman numeral in the main note string when you are arrowing along the notes.
